cwms-data-api/src/main/java/cwms/cda/api/timeseriesprofile/TimeSeriesProfileInstanceCatalogController.java

```diff
@@ -0,0 +1,104 @@
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package cwms.cda.api.timeseriesprofile;
+
+import static cwms.cda.api.Controllers.GET_ALL;
+import static cwms.cda.api.Controllers.LOCATION_MASK;
+import static cwms.cda.api.Controllers.OFFICE_MASK;
+import static cwms.cda.api.Controllers.PARAMETER_ID_MASK;
+import static cwms.cda.api.Controllers.STATUS_200;
+import static cwms.cda.api.Controllers.VERSION_MASK;
+import static cwms.cda.data.dao.JooqDao.getDslContext;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+import cwms.cda.data.dao.timeseriesprofile.TimeSeriesProfileInstanceDao;
+import cwms.cda.data.dto.timeseriesprofile.TimeSeriesProfileInstance;
+import cwms.cda.formatters.ContentType;
+import cwms.cda.formatters.Formats;
+import io.javalin.core.util.Header;
+import io.javalin.http.Context;
+import io.javalin.http.Handler;
+import io.javalin.plugin.openapi.annotations.HttpMethod;
+import io.javalin.plugin.openapi.annotations.OpenApi;
+import io.javalin.plugin.openapi.annotations.OpenApiContent;
+import io.javalin.plugin.openapi.annotations.OpenApiParam;
+import io.javalin.plugin.openapi.annotations.OpenApiResponse;
+import java.util.List;
+import javax.servlet.http.HttpServletResponse;
+import org.jetbrains.annotations.NotNull;
+import org.jooq.DSLContext;
+
+
+public final class TimeSeriesProfileInstanceCatalogController extends TimeSeriesProfileInstanceBase implements Handler {
+    public TimeSeriesProfileInstanceCatalogController(MetricRegistry metrics) {
+        tspMetrics(metrics);
+    }
+
+    @OpenApi(
+        queryParams = {
+            @OpenApiParam(name = OFFICE_MASK, description = "The office mask of the"
+                + " time series profile instance. Default is *"),
+            @OpenApiParam(name = LOCATION_MASK, description = "The location ID mask of the"
+                + " time series profile instance. Default is *"),
+            @OpenApiParam(name = PARAMETER_ID_MASK, description = "The parameter ID mask of the"
+                + " time series profile instance. Default is *"),
+            @OpenApiParam(name = VERSION_MASK, description = "The version mask of the"
+                + " time series profile instance. Default is *"),
+        },
+        method = HttpMethod.GET,
+        summary = "Get all time series profile instances. Masks can be provided to filter the results.",
+        tags = {TAG},
+        responses = {
+            @OpenApiResponse(status = STATUS_200,
+                description = "A TimeSeriesProfileInstance object",
+                content = {
+                    @OpenApiContent(from = TimeSeriesProfileInstance.class, type = Formats.JSONV1),
+                    @OpenApiContent(from = TimeSeriesProfileInstance.class, type = Formats.XMLV2),
```

Comment:
Only JSON is supported on new endpoints. make sure all controllers have this removed and there should be both JSONV1 and JSON so that you can choose either from swagger.

Reply:
Swapped instances of XMLV2 with JSON, added JSON to endpoints that needed it